adagrams/game.py

```diff
@@ -1,13 +1,129 @@
 from random import randint
+from .constants import LETTER_POOL, SCORE_CHART, ALPHABET
 
 def draw_letters():
-    pass
+
+    alphabet_pool = dict(LETTER_POOL)
+    results = []
+
+    while len(results) < 10:
+
+        random_num = randint(0, 25)
```

Comment:
Instead of hardcoding `25` here, consider using the length of `ALPHABET` minus 1.